taxi/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Car, Manufacturer
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"

```

Comment:
Form Field Exposure: Using `fields = "__all__"` can unintentionally expose sensitive fields. Specify only the fields you want to include to ensure data security.